### PR TITLE
grid_sample 

### LINKED ISSUE #21957: grid_sample

### PR DESCRIPTION
**Pull request by JONAHKYAGABA**
Close #21957
START ADDING GRID SAMPLE TO PADDLE FRONTEND

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by jkeane508** [comment]
> If you are working on an open task, please **edit** the PR description to link to the issue you've created.
> 
> For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).
> 
> Thank you 🤗

@JONAHKYAGABA Can you please reference the issue that you are tackling in this PR, Thanks!

**Comment by JONAHKYAGABA** [comment]
Close #21957

**Comment by JONAHKYAGABA** [comment]
Hi @jkeane508 sorry i had forgotten to link it but its sorted and am still updating the function.

File: ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_operators.py [hunk 1]
```diff
@@ -83,6 +83,15 @@ def test_jax_add(
         x=x[0],
         y=x[1],
     )
+@handle_frontend_test(
```
Comment (jkeane508) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_operators.py:86]: Hey, I don't think your test implementation is quite right. Please refer to the docs or take some inspiration from other tests in this file.

**Review by jkeane508 — APPROVED** [review]

File: ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_operators.py (hunk 1 above)
Comment (JONAHKYAGABA) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_operators.py]: Hello @jkeane508 thank you  for the guidance on this function but i made changes and i kindly request for a review,
And also am facing an error while running the test 
**" backend_version = {"version": paddle.version.full_version}
E   AttributeError: module 'paddle' has no attribute 'version'"**

**Comment by JONAHKYAGABA** [comment]
Hello @jkeane508 please kindly review this PR or inform me n how to proceed

**Pull request closed without merging**